setup.py

```diff
@@ -90,7 +90,7 @@ def run(self):
     "twitter": ["tweepy >= 3.5, < 4.0"],
     "dremio": ["pyarrow >= 5.0.0"],
     "exasol": ["pyexasol >= 0.16.1"],
-    "sodasql": ["soda-sql >= 2.0.0b25"],
+    "soda": ["soda-spark >= 0.2.1"],
```

Comment:
Removing the existing extra is a breaking change. You'll need to retain that. We can drop it in 1.0, although I'm curious why it's not needed anymore. `soda-spark` requires `soda-sql` I presume?

Reply:
Exactly, `soda-spark` requires `soda-sql`.
I'm going to revert back the change, but I agree we should drop it in 1.0.